frontend/src/components/ShootDetails/ShootInfrastructureCard.vue

```diff
@@ -25,9 +25,9 @@ limitations under the License.
           <v-icon color="cyan darken-2">cloud_queue</v-icon>
         </v-list-item-icon>
         <v-list-item-content>
-          <v-list-item-subtitle>Provider</v-list-item-subtitle>
+          <v-list-item-subtitle>{{vendorTitle}}</v-list-item-subtitle>
           <v-list-item-title class="pt-1">
-            {{shootCloudProviderKind}}
+            <vendor :shootItem="shootItem" :extended="true"></vendor>
```

Comment:
The vendorTitle and the vendor a coupled. Not things are separated into different components. I would keep one thing in one component.

Reply:
hm, the vender component is also used in the shoot list row, so I can't put
v-list-item-subtitle and v-list-item-title into the vendor component, or are you thinking of a different solution?

Reply:
I was not aware of the fact that the vendor component is used in the ShootListRow. We could add a boolean prop `title` and if true it renders the title
```js
      if (this.shootZones.length > 0) {  
         return `Provider / Region / ${this.zoneTitle}`
      }
      return 'Provider / Region'
```

Reply:
✅